lib/safira/store/buy.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Safira.Store.Buy do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Safira.Store.Redeemable
+  alias Safira.Accounts.Attendee
+
+  schema "buys" do
+    field :quantity, :integer
+
+    belongs_to :attendee, Attendee, foreign_key: :attendee_id, type: :binary_id
+    belongs_to :redeemable, Redeemable, foreign_key: :redeemable_id
+
+    timestamps()
+  end
+
+  def changeset(buy, attrs) do
+    buy
+    |> cast(attrs, [:attendee_id, :redeemable_id, :quantity])
+    |> validate_required([:attendee_id, :redeemable_id, :quantity])
+  end
```

Comment:
You'll probably update this changeset in a future PR but don't forget to add the needed validations.

Reply:
Yeah, the next PR will be focused on that logic. Anyway, I'm developing all the checks before running the changeset. I can run checks on the changeset and then it gets double checked. @joaop21

Reply:
The changeset's purpose is to validate the data. You don't need to double check, if you put the validations in the changeset and if something is is wrong the Repo operations won't persist it in DB.

Reply:
hmm, sounds good to me then! 👍